Doc/c-api/import.rst

```diff
@@ -204,6 +204,11 @@ Importing Modules
    Return the dictionary used for the module administration (a.k.a.
    ``sys.modules``).  Note that this is a per-interpreter variable.
 
+.. c:function:: PyObject* PyImport_GetModule(PyObject *name)
+
+   Return the already imported module with the given name.
```

Comment:
The only code that uses it (`_pickle_Unpickler_find_class_impl`) is written as `PyImport_GetModule` returning a borrowed reference.